### PR TITLE
Add code coverage reports

### PR DESCRIPTION
**Pull request by miguelandres**

### CODE REVIEW

File: remixer_storage/build.gradle [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 apply plugin: 'com.android.library'
 apply plugin: 'com.github.dcendents.android-maven'
+apply plugin: 'jacoco-android'
```
Comment (pingpongboss) [inline remixer_storage/build.gradle]: are you sure this works? How do you know? I don't see anything on https://codecov.io/gh/material-foundation/material-remixer-android

See my description of http://codereview.cc/D1777

File: remixer_ui/build.gradle [hunk 2]
```diff
@@ -16,13 +16,19 @@
 
 apply plugin: 'com.android.library'
 apply plugin: 'com.github.dcendents.android-maven'
+apply plugin: 'jacoco-android'
```
Comment (pingpongboss) [inline remixer_ui/build.gradle]: same with this android library.

**Comment by pingpongboss** [comment]
Also see http://stackoverflow.com/questions/28586882/jacoco-and-unit-tests-code-coverage-with-android-gradle-plugin-1-1/

File: .travis.yml [hunk 3]
```diff
@@ -24,7 +24,7 @@ licenses:
   - 'android-sdk-license-.+'
 
 script:
-  - ./gradlew check
+  - ./gradlew check jacocoTestReport
```
Comment (pingpongboss) [inline .travis.yml:27]: check should already include jacocotestreport right?

File: remixer_storage/build.gradle (hunk 1 above)
Comment (miguelandres) [reply remixer_storage/build.gradle]: Fixed, You're right.

File: remixer_ui/build.gradle (hunk 2 above)
Comment (miguelandres) [reply remixer_ui/build.gradle]: Fixed, You're right.

File: .travis.yml (hunk 3 above)
Comment (miguelandres) [reply .travis.yml:27]: It doesn't, I tried locally and I didn't  see it run :/

**Review by pingpongboss — APPROVED** [review]

**Pull request merged**